src/revpi-functions

```diff
@@ -9,3 +9,12 @@ cm1-detection() {
     grep -m 1 "model name" /proc/cpuinfo | grep -q v6l
     return $?
 }
+
+# Check existance of HAT EEPROM according to document:
+# https://github.com/RevolutionPi/revpi-hat-eeprom/blob/master/RevPi-HAT-EEPROM-Format.md#custom-atoms
+has-hat-eeprom() {
+	if [ -e /proc/device-tree/hat/custom_0 ]; then
```

Comment:
Just check for custom_1 then you don't need to worry that when you read the file it is not there. And it is required the same as custom_0. But this doesn't change much.